tests/html/tiny_test.html

```diff
@@ -1 +1 @@
-  
```

Comment:
Correction, it was created in a3225428250a559c25af09f442e07e504d6e49b5 and the trailing space existed then, too. It's fine to remove it at this point.